src/goesproc/handler_goesr.cc

```diff
@@ -98,6 +98,29 @@ void GOESRImageHandler::handle(std::shared_ptr<const lrit::File> f) {
   fb.region = details.region;
   fb.channel = details.channel;
 
+  // Process image data function
```

Comment:
Can you add a short description or sample data of the contents of this field, for reference.